src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSchedulingAndDescription/index.spec.tsx

```diff
@@ -0,0 +1,349 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
Same as in the other PR. The tests in this file are really slow. You have to investigate why, the CI is failing with a timeout error